docs/how-to-guides/editable-installs.rst

```diff
@@ -0,0 +1,79 @@
+.. _how-to-guides-editable-installs:
+
+*****************
+Editable installs
+*****************
+
+We have support for editable installs! To use it, simply pass the
+``-e``/``--editable`` option to pip_.
+
+
+.. code-block:: console
+
+   $ pip install -e .
+
+
+This will install the project in the current directory in editable mode. When a
+``meson-python`` package is installed in editable mode, it will be re-built the
+first time it is imported. That means, the first time you import a module from
+your package, it will take a bit longer.
+
+
+.. admonition:: What is the scope of editable installs?
+   :class: seealso
+
+   The editable installs feature only targets the Python modules of the project.
+   Metadata-based features, like `entry points`_, and similar mechanisms will
+   not be updated when using editable installs. This is a limitation of editable
+   installs specification itself.
+
+
+.. _how-to-guides-editable-installs-verbose:
+
+Verbose mode
+============
+
+It might be useful for you to see the output of Meson_ when the package is being
+re-built. We provide a *verbose mode* that does this, and can be enabled either
+temporarily, after the install, or permanently, in the install.
+
+
+Enable it temporarily
+---------------------
+
+To enable the verbose mode on any existing ``meson-python`` editable install,
+you simply need to set the ``MESONPY_EDITABLE_VERBOSE`` environment variable
+to any non-null value.
+
+
+.. code-block:: console
+
+   $ MESONPY_EDITABLE_VERBOSE=1 python
+
+
+Enable it permanently
```

Comment:
Just deleting these section headers will make the page look better. If it's one sentence + one line of code, that's a sign that it's too little for its own section.

Reply:
This applies to other pages as well. For example, the _Use build config settings_ does not need section headers for `pypa/build` and `pip`.

Reply:
Yeah :sweat_smile:. I wanted to use smaller headers, but rST doesn't allow that.